doc/nrf/device_guides/working_with_nrf/nrf70/features.rst

```diff
@@ -9,21 +9,18 @@ Features of nRF70 Series
 
 The nRF70 Series of wireless companion ICs add Wi-Fi® capabilities to another System-on-Chip (SoC), Microprocessor Unit (MPU), or Microcontroller (MCU) host device.
 It implements the Physical (PHY) and Medium Access Control (MAC) layers of the IEEE 802.11 protocol stack, while the higher layers of the networking stack run on the host device.
-The nRF70 series wireless companion IC includes the RF front end and power management function.
+The nRF70 Series wireless companion IC includes the RF front end and power management function.
 
-The nRF70 series devices are designed for Internet of Things (IoT) applications and are ideal for adding modern Wi-Fi 6 capabilities to existing Bluetooth® Low Energy, Thread, or Zigbee systems, as well as adding Wi-Fi Access Point scanning capabilities to LTE/GPS systems.
+The nRF70 Series devices are designed for Internet of Things (IoT) applications and are ideal for adding modern Wi-Fi 6 capabilities to existing Bluetooth® Low Energy, Thread, or Zigbee systems, as well as adding Wi-Fi Access Point scanning capabilities to LTE/GPS systems.
 
 For additional information, see the following documentation:
 
+* `nRF70 Series`_ for the technical documentation on the nRF70 Series devices.
+* :ref:`nrf7002dk_nrf5340`.
+* :ref:`ug_nrf70_developing` documentation for more advanced topics related to the nRF70 Series.
 * :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.
-* :ref:`nrf7002dk_nrf5340`
 
-The following figure illustrates the partitioning of the Wi-Fi stack between the nRF70 series device and the host device:
```

Comment:
The protocol stack figure https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/wifi/wifi.html#wi-fi-protocol-stack, is there in "Wi-Fi overview", which is similar to the deleted one except for the partitioning of the Wi-Fi stack between the nRF70 series device and the host device, hence deleted.

Let me know if we intend to keep both.

Reply:
Thanks for pointing out the difference to me @richabp. My thinking:
- the partitioning of the Wi-Fi stack between the host and the companion chip is specific to the nRF70 Series, and I think we should keep it (the figure and the corresponding text) in this Chapter. It really helps because we do not explain stack partitioning anywhere else. We can explain it here, and refer to it from other parts of the documentation.
- for the above reason, this part should be made into a specific sub-section. An example of title would be "Networking stack architecture and partitioning"
- we seem to have two options, for hosting this subsections: keep it in "Features" or move it into the last part: "Developing with nRF70 Series". I favor the second option but OK with both.

Reply:
Based on offline discussion with @ioannisg added a new page "Networking stack partitioning" in the "developing with.." guide.

Let me know if there is something else that needs to be added on this page.

Reply:
Looks good to me.